src/syrupy/utils.py

```diff
@@ -14,3 +19,22 @@ def walk_snapshot_dir(root: str) -> Generator[str, None, None]:
             continue
         if not filepath.name.startswith(".") and filepath.is_file():
             yield str(filepath)
+
+
+def import_module_member(path: str) -> Any:
```

Comment:
`Any` or can we be specific and reference a concrete implementation of `AbstractSyrupyExtension` as we do here:  https://github.com/tophat/th_snapshot/blob/331c5d5e85310630bd55a101e3bd4744f51c770d/src/syrupy/assertion.py#L57